deploy.sh

```diff
@@ -24,10 +25,17 @@ if [ "$registry" != "remote" ] && [ "$registry" != "local" ]; then
     exit 2
 fi
 
-if cd deploy/$1 && kubectl apply -f crd.yaml && kubectl apply -f rbac.yaml && kubectl apply -f deploy_$registry.yaml; then
+if [ "$app" == "v_stateful_set" ]; then
+    app_ns="vstatefulset"
+else
+    app_ns=$app
+fi
+
+## use imperative management for CRDs since metadata for PodTemplateSpec is too long.
+if cd deploy/$1 && kubectl create -f crd.yaml && kubectl apply -f rbac.yaml && kubectl apply -f deploy_$registry.yaml; then
```

Comment:
This change is necessary since `VStatefulSetSpec` contains a field `template` of type `PodTemplateSpec`, whose metadata makes the resulting CRD file too big for the declarative API.